templates/emails/email_base.html

```diff
@@ -0,0 +1,42 @@
+{% load static %}
+{% load inlinecss %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="Content-Type" content="text/html charset=UTF-8"/>
+    <title>Confirm your email address</title>
+    <style>
+        @import url('https://fonts.googleapis.com/css?family=Chivo:400,400i,700|Playfair+Display:400,700&display=swap');
+    </style>
+</head>
+{% inlinecss "css/emails.css" %}
+    <body>
+    <div class="main">
+        <img class="opencraft-logo" src="{{ base_url }}{% static 'img/svg/logo-for-email.svg' %}"
+             alt="Opencraft Logo" aria-hidden="true">
+        <div class="main">
+            <img src="{{ base_url }}{% static 'img/jpg/header-background.jpg' %}" alt="Email Header" aria-hidden="true">
+            <div class="content">
+                {% block content %}
+                {% endblock %}
+            </div>
+            <div class="addendum">
+                <p>
+                    And if you need anything, or have any questions
+                    or remarks, don't hesitate to reply to this email.
+                </p>
+            </div>
+        </div>
+        <div class="footer">
+            <p>
+                <a href="{{ unsubscribe_url }}">Unsubscribe</a> | edX and Open edX are registered trademarks of edX Inc.
```

Comment:
I think this is more of a TODO item for now. Perhaps I should add an explicit TODO here. 
Similar to the instance customisation link. If we have a link to the profile edit page this can point to that since we'll have an option to disable update emails. 

Or we can remove this entirely since the emails right now are triggered by user actions and are not a subscription.

Reply:
> Or we can remove this entirely since the emails right now are triggered by user actions and are not a subscription.

It's fine to leave it with a TODO here :+1: 
We might need to add it later to some emails anyway.